src/lib/pages/backoffice/Providers/ProviderSearch/ProviderListEntry.js

```diff
@@ -115,7 +116,9 @@ export default class ProviderListEntry extends Component {
             data-test={`navigate-${providerMetadata.pid}`}
           >
             <ProviderIcon />
-            {providerMetadata.name}
+            <Truncate lines={2} width="500px">
```

Comment:
Can we do %age instead of pixels? It'll be more responsive to different width screens then.